packages/client-twitter/src/base.ts

```diff
@@ -218,74 +218,67 @@ export class ClientBase extends EventEmitter {
             count
         );
-
-        // console.dir(homeTimeline, { depth: Infinity });
-
         return homeTimeline.tweets;
-        // .filter((t) => t.__typename !== "TweetWithVisibilityResults")
-        // .map((tweet) => {
-        //     // console.log("tweet is", tweet);
-        //     const obj = {
-        //         id: tweet.id,
-        //         name:
-        //             tweet.name ??
-        //             tweet. ?.user_results?.result?.legacy.name,
-        //         username:
-        //             tweet.username ??
-        //             tweet.core?.user_results?.result?.legacy.screen_name,
-        //         text: tweet.text ?? tweet.legacy?.full_text,
-        //         inReplyToStatusId:
-        //             tweet.inReplyToStatusId ??
-        //             tweet.legacy?.in_reply_to_status_id_str,
-        //         createdAt: tweet.createdAt ?? tweet.legacy?.created_at,
-        //         userId: tweet.userId ?? tweet.legacy?.user_id_str,
-        //         conversationId:
-        //             tweet.conversationId ??
-        //             tweet.legacy?.conversation_id_str,
-        //         hashtags: tweet.hashtags ?? tweet.legacy?.entities.hashtags,
-        //         mentions:
-        //             tweet.mentions ?? tweet.legacy?.entities.user_mentions,
-        //         photos:
-        //             tweet.photos ??
-        //             tweet.legacy?.entities.media?.filter(
-        //                 (media) => media.type === "photo"
-        //             ) ??
-        //             [],
-        //         thread: [],
-        //         urls: tweet.urls ?? tweet.legacy?.entities.urls,
-        //         videos:
-        //             tweet.videos ??
-        //             tweet.legacy?.entities.media?.filter(
-        //                 (media) => media.type === "video"
-        //             ) ??
-        //             [],
-        //     };
-        //     // console.log("obj is", obj);
-        //     return obj;
-        // });
     }
 
-    async fetchFeedTimeline(count: number): Promise<string> {
+    async fetchHomeTimeline(count: number): Promise<Tweet[]> {
         elizaLogger.debug("fetching home timeline");
         const homeTimeline = await this.twitterClient.fetchHomeTimeline(count, []);
-        return homeTimeline
-            .filter(tweet => tweet.text || tweet.legacy?.full_text)
-            .sort((a, b) => {
-                const timestampA = new Date(a.createdAt ?? a.legacy?.created_at).getTime();
-                const timestampB = new Date(b.createdAt ?? b.legacy?.created_at).getTime();
-                return timestampB - timestampA;
-            })
-            .slice(0, count)
-            .map(tweet =>
-                `@${tweet.username || tweet.core?.user_results?.result?.legacy?.screen_name}: ${tweet.text ?? tweet.legacy?.full_text ?? ''}`
-            )
-            .join('\n');
+
+        //console.dir(homeTimeline, { depth: Infinity });
```

Comment:
Should the `console.log` be removed or should we make them `elizaLogger.debug()` for debugging to avoid uncommenting and rebuilding?

Reply:
Not my code, was pre-existing. We don't have an equivalent of .dir in elizaLogger afaik